devtools/azure_ci_images.md

```diff
@@ -0,0 +1,231 @@
+ACR can only be done if the following conditions are met:
+
+* The user with access to the ACR is considered an "Owner" by the ACR security, this permission can only be 
+  authorized by the ACR administrator
+* The user with access to the ACR is a member of the Azure DevOps Pipeline team which is wanting to queue up the build.
+* The user with access to the ACR is an Administrator on the DevOps Pipeline team
+* The user with access to the ACR is signed into DevOps and approves using their credentials for the Pipeline team 
+  to access the ACR.
+* The user with access to the ACR approves specific builds to have access to the ACR. 
+
+Then, and only then, can a Pipline build access images in the ACR. Further, no unauthorized user can see, manage, 
+or change these credentials.
+
+## Authenticating a new users for ACR and Pipelines
+
+These are instructions for allowing a new user permission to use the private ACR instance paid for by MolSSI/VT. These 
+are a one time thing and if you cannot meet these criteria, skip down to section on "Adding a new code to the CI" and 
+coordinate with one of the approved users who can upload your custom Docker image (or help you make one) to the private 
+ACR.
+
+1. Have an [Azure Portal](https://azure.microsoft.com/en-us/account/) account through a Virginia Tech email address 
+   (this is something which has to be authorized by VT so we can allow access to the ARC, this may be relaxed in the 
+   future.)
+2. Create an [Azure DevOps](https://azure.microsoft.com/en-us/services/devops/?nav=min) account with the same email 
+   address. These are different login nodes, even though they both run through Azure. This will be for access to the 
+   Pipelines team.
+3. Request access to the QCArchive DevOps team. This will require reaching out to one of the QCArchive developers.
+4. Request access to the QCArchive ACR. This will require reaching out to one of the QCArchive developers, this is 
+   also a smaller group.
+
+*If you only want to upload images to the ACR (i.e. the Pipeline is already connected to the ACR)*
+
+* Request that you be made a "Contributor" in the ACR administration.
+
+*If you want to manage account connections between the ACR and Pipelines*
+
+* Request that you be made an "Owner" in the ACR administration.
+* Request that you be made an "Administrator" in the Pipelines team.
+
+## Connecting ACR to Pipelines
+
+See also (its missing details about the security setup)
+
+https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml
+
+*Note: This only has to be done once per ACR (not per image) and Pipelines Team (not per build).*
+
+### Pre-requisites
+
+* Do everything in the previous section
+* Be made an "Owner" of ACR and an "Administrator" of the Pipelines team
+* Be signed into the account which meets these criteria.
+
+### Linking the services
+
+1. Sign into the account which meets the pre-requisites through the DevOps login.
```

Comment:
`0.`

Reply:
Markdown doesn't care about numbered list ordering. I could have just put `1. 1. 1. 1111. 1. 11.` and it would have ordered them correctly in render. It does look a bit odd to the author though. I fixed it anyways.